src/human-readable/runtime/cache.ts

```diff
@@ -1,5 +1,8 @@
 import type { AbiParameter } from '../../abi'
 
+export function getParameterCacheKey(param: string, type?: string) {
+  return `${type ? `${type}:` : ''}${param}`
+}
 export function createAbiParameterCache() {
```

Comment:
Let's keep this named `createParameterCache`:

```suggestion
export function createParameterCache() {
```

Also, any reason why this a function instead of constant value? `const parameterCache = new Map<…>(…)`

Reply:
No particular reason. I mostly though it would look nicer and it would be more explicit for people reading the code.